core/senders/qq.py

```
@@ -522,10 +522,22 @@ async def send(
             )
 
         context_target_sessions = self._resolve_context_target_sessions(qq_targets)
+        real_batches = self._flatten_batches(batches)
 
-        if not context_target_sessions:
+        if not real_batches:
+            logger.debug("[QQSender] 展平后无有效批次，跳过发送")
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)